src/llm_vm/guided_completion.py

```diff
@@ -0,0 +1,20 @@
+import outlines.models as models 
+import outlines.text.generate as generate
+
+model = models.transformers("gpt2")
+class RegexCompletion:
```

Comment:
could you add some documentation to what these are

Reply:
Sure will add some comments